libcaf_core/caf/detail/parse.hpp

```diff
@@ -43,30 +43,40 @@ struct literal {
   std::string_view str;
 };
 
+/// @memberof caf::detail
```

Comment:
Why is this necessary? The function is properly defined in the namespace.

Reply:
It seems like doxygen can not parse the namespace and function properly without this command. I am looking into the cause in more details and prepare a separate PR for it.